xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-test/xwiki-platform-office-test-docker/pom.xml

```diff
@@ -48,10 +24,25 @@
          the "officemanager" ScriptService which requires PR and that pages is executed in the context of
          XWiki.XWikiPreferences (which is the current doc) -->
     <xwikiPropertiesAdditionalProperties>test.prchecker.excludePattern=.*:XWiki\.XWikiPreferences</xwikiPropertiesAdditionalProperties>
+    <!-- START: Required to fix conflicts resolution between XWiki & TestContainers -->
+    <!-- selenium-server requires Guava 25.0-jre+ but XWiki uses Guava 20.0, see xwiki-commons top level POM for more
+      details as to why we cannot use a more recent version. Thus we need to force a more recent version of Guava
+      here. Note that the version needs to be checked when we change the version of selenium-java -->
+    <guava.version>25.0-jre</guava.version>
+    <!-- END: Required to fix conflicts resolution between XWiki & TestContainers -->
```

Comment:
why do you need a start/end when there's only 1 property related to guava?

Reply:
Copy/Paste from what's in menu-test-docker-test pom.xml